scripts/calculator.js

```diff
@@ -0,0 +1,121 @@
+import { Expression } from "./expression.js";
+import { History } from "./history.js";
+
+class Calculator {
+    constructor (calculatorSelector, inputFieldSelector, buttonSelectors) {
+        // DOM Selectors
+        this.calculatorContainer = document.querySelector(calculatorSelector);
+        this.inputField = document.querySelector(inputFieldSelector);
+        this.buttons = document.querySelectorAll(buttonSelectors);
+
+        this.currentInput = "";
+        this.resultDisplayed = false;
+        this.history = [];
+
+        this.expression = new Expression();
+        this.h = new History(this.history, "history");
+        
+        this.init();
+    }
+
+    init () {
+        this.buttons.forEach(button => {
+            button.addEventListener("click", (e) => this.handleButtonClick(e));
+        });
+
+        document.addEventListener("keydown", (e) => this.handleKeyEvent(e));
+    }
+
+    // for update input field values
+    updateInputField (value) {
+        if (this.resultDisplayed && /[0-9.]/.test(value)) {
+            this.clearInputField();
+        }
+        this.resultDisplayed = false;
+        this.currentInput += value;
+        this.inputField.textContent = this.currentInput;
+        this.inputField.scrollTo(this.inputField.offsetWidth, 0);
+    }
+
+    // for clear eniter input field
+    clearInputField () {
+        this.currentInput = "";
+        this.inputField.textContent = this.currentInput;
+    }
+
+    // for remove only last charecter
```

Comment:
Minor Nit: typo in spelling and no need to add `for`, just describe the functionality. Please make the changes to other places as well.
```suggestion
    // Removes only the last character
```

Reply:
I updated my comments (https://github.com/krushna-simform/scientific-calculator/pull/3/commits/aa0b549cd010135ccb11b9fffb1aed33c0b236c2)